recipes/gql/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "gql" %}
+{% set version = "0.1.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "d20cf1eb1c6d54558174a3dcc3cf1ca0e07678cb1123804c2e165838238350f4" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six >=1.10.0
```

Comment:
Are these really build requirements? Can you try to build without them? (`conda` will install the run requirements for the users at installation time)

Reply:
I tried and it worked on my Mac. I tried again on a linux machine and it failed. I'm keeping them for now.

Reply:
Setuptools is fine, I specifically meant the requirements other than `python` and `setuptools` (six, graphql and promise). I ask because they don't seem needed to run the `setup.py`

Reply:
Should I try without? Travis is experiencing huge delays, I don't know what to do.

Reply:
Since this is a noarch package we don't need travis (or appveyor) to pass to merge, once it's good on circle I can merge it.

Reply:
So would you mind trying it without? (You can also try to build it on your local and see if that works before commiting/pushing)

Reply:
Yes. I did that and I already pushed the commit. It already passed on CircleCI.